Modix/ModixBot.cs

```diff
@@ -53,30 +56,25 @@ public ModixBot(ModixConfig config, ILogger logger)
             });
 
             await Install(); // Setting up DependencyMap
+
             _map.AddDbContext<ModixContext>(options =>
             {
                 options.UseNpgsql(_config.PostgreConnectionString);
             }, ServiceLifetime.Transient);
 
             _host = ModixWebServer.BuildWebHost(_map, _config);
 
-            //disable until we migrate to Xero's host.
-            //#if !DEBUG
-
-            //#endif
-
             _scope = _host.Services.CreateScope();
 
             using (var context = _scope.ServiceProvider.GetService<ModixContext>())
             {
                 context.Database.Migrate();
             }
 
-            using (var context = _scope.ServiceProvider.GetService<ModixContext>())
-            {
-                context.ChannelLimits.ToList();
-            }
+            var configurationService = _scope.ServiceProvider.GetService<IBehaviourConfigurationService>();
```

Comment:
Should be `GetRequiredService<T>()` since we definitely assume it will be available on the next line.

Reply:
Didn't know that was a thing, TIL